github/dependabot_secrets.go

```diff
@@ -144,8 +145,29 @@ func (s *DependabotService) CreateOrUpdateRepoSecret(ctx context.Context, owner,
 //
 // GitHub API docs: https://docs.github.com/en/rest/dependabot/secrets#create-or-update-an-organization-secret
 func (s *DependabotService) CreateOrUpdateOrgSecret(ctx context.Context, org string, eSecret *DependabotEncryptedSecret) (*Response, error) {
+	repoIDs := make([]string, len(eSecret.SelectedRepositoryIDs))
+	for i, secret := range eSecret.SelectedRepositoryIDs {
+		repoIDs[i] = strconv.Itoa(int(secret))
+	}
+	params := struct {
+		*DependabotEncryptedSecret
+		SelectedRepositoryIDs []string `json:"selected_repository_ids,omitempty"`
+	}{
+		DependabotEncryptedSecret: eSecret,
+		SelectedRepositoryIDs:     repoIDs,
+	}
+
+	putSecret := func(ctx context.Context, url string, eSecret interface{}) (*Response, error) {
+		req, err := s.client.NewRequest("PUT", url, eSecret)
+		if err != nil {
+			return nil, err
+		}
+
+		return s.client.Do(ctx, req, nil)
+	}
```

Comment:
Created this inline for now as it's the only place it's used. Happy to pull it out as an unexported function to match the `putSecret` package function (maybe `putOrgSecret`). That would require though defining a package type for `params` (for used as an input argument). I thought it was an overkill for now, so left it as such.

Reply:
There's no need for the inline function here.  Let's just please delete lines 160 and 167, and move lines 161-166 down to replace line 170 below.

Reply:
Sounds good - was trying to keep the symmetry with what was previously there. Removed!